Original patch from: ../crosstool-NG/ct-ng.trunk/patches/binutils/2.19/170-use-relro.patch

-= BEGIN original header =-
Original patch from Gentoo:
gentoo/src/patchsets/binutils/2.19/78_all_use-relro.patch

-= END original header =-

diff -durN binutils-2.19.1.orig/ld/ldmain.c binutils-2.19.1/ld/ldmain.c
--- binutils-2.19.1.orig/ld/ldmain.c	2009-03-08 11:57:03.000000000 +0100
+++ binutils-2.19.1/ld/ldmain.c	2009-03-08 11:57:03.000000000 +0100
@@ -264,6 +264,7 @@
   link_info.keep_memory = TRUE;
   link_info.new_dtags = TRUE;
   link_info.combreloc = TRUE;
+  link_info.relro = TRUE;
   link_info.strip_discarded = TRUE;
   link_info.emit_hash = TRUE;
   link_info.callbacks = &link_callbacks;
